packages/datatrak-web/src/features/Questions/EntityQuestion/utils.ts

```diff
@@ -4,14 +4,22 @@
  */
 import { useUser } from '../../../api/queries';
 import { useSurveyForm } from '../../Survey/SurveyContext';
+import { DatatrakWebSubmitSurveyRequest as RequestT } from '@tupaia/types';
 
-export const useEntityBaseFilters = config => {
+export const useEntityBaseFilters = (config: RequestT.EntityQuestionConfig) => {
```

Comment:
This type for the EntityQuestionConfig was initially in the front-end but I wanted to use it in the server to so I moved it to the request type. I'm still not sure if that's the correct place for it.

Reply:
I think that's fine, and probably the most sensible approach